src/dropdown/_dropdown.scss

```diff
@@ -43,7 +43,7 @@ $dropdown-header-horizontal-padding: 8px;
   width: 100%;
   height: var(--dropdown-header-button-height);
```

Comment:
I think we can define these via `--button-width` and `--button-height`, can we ?

Reply:
Agreed. Let's do it in another PR